### PR TITLE
Add database access manager schema

### PR DESCRIPTION
**Pull request by janboll**
Implements: [APPSRE-8131](https://issues.redhat.com/browse/APPSRE-8131)

### CODE REVIEW

File: schemas/app-interface/database-access-1.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-interface/database-access-1.yml
+  name:
+    type: string
+  username:
+    type: string
```
Comment (chassing) [inline schemas/app-interface/database-access-1.yml:15]: type `string` can contain spaces, and I assume spaces and other characters aren't allowed in a username. please choose another type

File: schemas/app-interface/database-access-1.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  database:
+    type: string
```
Comment (chassing) [inline schemas/app-interface/database-access-1.yml:17]: same as with username

File: schemas/app-interface/database-access-1.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        grants:
+          type: array
+          items:
+            type: string
+        dbschema:
+          type: string
```
Comment (chassing) [inline schemas/app-interface/database-access-1.yml:31]: same as with username

File: schemas/app-interface/database-access-1.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  access:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        grants:
+          type: array
+          items:
+            type: string
```
Comment (chassing) [inline schemas/app-interface/database-access-1.yml:29]: Does it make sense to have an `enum` with all possible grants here?

File: schemas/app-interface/database-access-1.yml (hunk 1 above)
Comment (janboll) [reply schemas/app-interface/database-access-1.yml:15]: Good point! I addressed this

File: schemas/app-interface/database-access-1.yml (hunk 4 above)
Comment (janboll) [reply schemas/app-interface/database-access-1.yml:29]: I added an enum, can you please check?

Comment (janboll) [reply schemas/app-interface/database-access-1.yml:30]: Actually, wait, I have to review the permissions again

**Review by chassing — APPROVED** [review]

**Pull request merged**